tests/test_dagmc_query.py

```diff
@@ -68,3 +80,12 @@ def test_get_tris_dimension_incorrect():
             if 'Meshset is not a volume nor a surface! Rootset will be used by default.' in str(w[-1].message):
                 test_pass[1] = True
     assert(all(test_pass))
+
+def test_calc_tris_per_vert():
```

Comment:
Can we add a test for the case where a meshset is given when the `DagmcQuery` is created and confirm the expected behavior?